app/views/pages/board.html.erb

```diff
@@ -0,0 +1,90 @@
+<table id = "chess_board"> 
+<tr> 
```

Comment:
There is a lot of repetition here. Maybe use a set of loops to generate the rows/cols instead?